execution/execution-correctness/src/execution_correctness_manager.rs

```diff
@@ -2,25 +2,48 @@
 // SPDX-License-Identifier: Apache-2.0
 
 use crate::{
-    local_client::LocalClient,
+    execution_correctness::ExecutionCorrectness,
+    local::{LocalClient, LocalService},
     process::ProcessService,
     remote_service::RemoteService,
     serializer::{SerializerClient, SerializerService},
     spawned_process::SpawnedProcess,
     thread::ThreadService,
 };
 use executor::Executor;
-use executor_types::BlockExecutor;
 use libra_config::config::{ExecutionCorrectnessService, NodeConfig};
+use libra_crypto::ed25519::Ed25519PrivateKey;
+use libra_global_constants::EXECUTION_KEY;
+use libra_secure_storage::{Storage, Value};
 use libra_vm::LibraVM;
 use std::{
+    convert::TryInto,
     net::SocketAddr,
     sync::{Arc, Mutex},
 };
 use storage_client::StorageClient;
 
+pub fn extract_execution_prikey(config: &mut NodeConfig) -> Ed25519PrivateKey {
```

Comment:
this diverges from the safety rules approach, the big concern here is that in the real final implementation we'll have
1) key for signing for lsr
2) key for authenticated channel with libra-node
and who knows what else we decide might make sense at some point in time here or whether or not we want to think about rotation, would recommend we just adopt passing in a storage solution instead.

Now if you want to move fast, this seems okay, but we really need to get this documented in our startup process for config/management, which I haven't yet seen in this PR how you do. But documentation goes beyond github docs.